packages/utils/src/types/config.ts

```diff
@@ -47,12 +47,16 @@ export type IgnoredUrl = {
     hints: string[];
 };
 
+export type HintConfigArray = [string, { [key: string]: any }];
```

Comment:
Do we need to export these types? While we are here maybe we can do some cleaning?

Reply:
It is not needed right now, we are not referencing it anywhere. Do you want me to remove the export?

Reply:
Yes, and any other not used elsewhere if you can.
Thanks!